src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -314,8 +314,9 @@ class HoldRequest extends React.Component {
     const itemSource = selectedItem.itemSource;
     const deliveryLocations = this.props.deliveryLocations;
     const isEddRequestable = this.props.isEddRequestable;
+    const allClosed = closedLocations.includes('');
```

Comment:
So, all locations are closed if there's an empty string in the list of locations? I thought `'all'` was the option for all closed? If `CLOSED_LOCATIONS` is empty, the fallback is an empty array. Might it be better to check if `closedLocations` has a length of 0 instead of it including an empty string?